raylibc/rcore.c

```diff
@@ -124,8 +124,8 @@
 #define RLGL_IMPLEMENTATION
 #include "rlgl.h"                   // OpenGL abstraction layer to OpenGL 1.1, 3.3+ or ES2
 
-#define RAYMATH_IMPLEMENTATION      // Define external out-of-line implementation
-#include "raymath.h"                // Vector3, Quaternion and Matrix functionality
+//#define RAYMATH_IMPLEMENTATION      // Define external out-of-line implementation
+//#include "raymath.h"                // Vector3, Quaternion and Matrix functionality
```

Comment:
I was going to say "hey, how does this work?" But I realized it's because the rcore module has been fully ported to D! Anything that would have used raymath is commented out.

So this is pretty nice.